packages/api/src/command/medical/cq-directory/update-cq-directory-entry.ts

```diff
@@ -0,0 +1,34 @@
+import { CQDirectoryEntry, CQDirectoryEntryData } from "../../../domain/medical/cq-directory";
+import { validateVersionForUpdate } from "../../../models/_default";
+import { CQDirectoryEntryModel } from "../../../models/medical/cq-directory";
+import { executeOnDBTx } from "../../../models/transaction-wrapper";
+import { BaseUpdateCmd } from "../base-update-command";
+import { getCQDirectoryEntryOrFail } from "./get-cq-directory-entry";
+
+export type CQOrganizationUpdateCmd = BaseUpdateCmd & CQDirectoryEntryData;
+
+export const updateCQDirectoryEntry = async (
+  cqOrganizationUpdate: CQOrganizationUpdateCmd
+): Promise<CQDirectoryEntry> => {
+  const { eTag, name, urlXCPD, urlDQ, urlDR, lat, lon, state, data } = cqOrganizationUpdate;
+
+  return executeOnDBTx(CQDirectoryEntryModel.prototype, async transaction => {
+    const org = await getCQDirectoryEntryOrFail({ oid: cqOrganizationUpdate.oid });
+
+    validateVersionForUpdate(org, eTag);
+
+    return org.update(
```

Comment:
Executing stuff like this on a DB tx more expensive than just issuing a single update.

Any particular reason to do so? In theory, once we start doing operations with an entity on DB tx, all operations should follow suit.

Another point is the fact this will be updating A LOT of records daily, right? So even more important to be sure we need DB txs for this one.

Lastly, if we do need DB tx, (in most situations) it only works if all DB operations are within the DB tx, so the load should also use the `transaction`.

Reply:
I think this might actually not require the DB tx.. 🤔

Reply:
Thx for raising this point and explaining it!